### DISCUSSION TITLE
Running unit tests requires pytorch

### DISCUSSION THREAD

**Discussion by Dexterp37** [fcakyon/labelme2coco#21] [General]
Hey!

Unit test for this project require pytorch to run but pytorch is not advertised as a dependency anywhere in [the requirements file](https://github.com/fcakyon/labelme2coco/blob/master/requirements.txt). This is due to this [line in sahi](https://github.com/obss/sahi/blob/cbe6fd707356137a773654819958ce9f62080a2a/sahi/__init__.py#L5) which gets triggered by [this line](https://github.com/fcakyon/labelme2coco/blob/037b60fb6867760848bc24434dbee55866aaf89d/tests/test_unit.py#L25).

It feels like an overkill to require pytorch for these unit tests. What do you think?

**Comment by fcakyon** [comment]
You can run the tests over github actions after opening a PR, instead of testing in local. Would that solve your issues?
